### DISCUSSION TITLE
Is it possible too run a different wallpaper on your second screen?

### DISCUSSION THREAD

**Discussion by Boaztheostrich** [rocksdanister/lively#572] [Q&A] [answered]
I think that the option too use two seperate wallpapers would really put this application right in line with wallpaper engine although the price has already made it surpass it for me.

**Comment by rocksdanister** [comment, marked as answer]
![Screenshot 2021-06-22 092738](https://user-images.githubusercontent.com/17554161/122860874-1f350900-d33c-11eb-9111-08879d56a691.png)

**Reply by Boaztheostrich** [reply]
That didn't work for me, could you possibly elaborate?

**Reply by rocksdanister** [reply]
https://youtu.be/8vgrsoBGoHI?t=19

**Reply by Boaztheostrich** [reply]
Oh I got it now thanks!!!! Really appreciate the quick response

**Comment by MalungoAlberto** [comment]
@rocksdanister  Is it possible to change only the backgound to be a Picture but the animation remains???

**Reply by rocksdanister** [reply]
Not possible.
